tests/NotPresentRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Saritasa\Laravel\Validation\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Saritasa\Laravel\Validation\Rule;
+
+class NotPresentTest extends TestCase
+{
+    /**
+     * Test "not_present" rule
+     */
+    public function testNotPresent()
+    {
+        $this->assertEquals('not_present', Rule::notPresent());
+    }
+
+    /**
+     * Test "not_present" rule with mixed usage
+     */
+    public function testMixedForNotPresent()
+    {
+        $rules = Rule::string()->notPresent()->required()->min(3);
+        $this->assertEquals('string|not_present|required|min:3', $rules);
```

Comment:
I know, that this rule is for test purposes only, but how it is possible that attribute should not be presented and should be required?